alacritty/Cargo.toml

```diff
@@ -56,22 +56,3 @@ wayland = ["alacritty_terminal/wayland"]
 live-shader-reload = []
 nightly = []
 bench = []
-
-[package.metadata.deb]
-maintainer = "Christian Duerr <contact@christianduerr.com>"
-license-file = ["../LICENSE-APACHE", "3"]
-extended-description = """\
-Alacritty is the fastest terminal emulator in existence. Using the GPU for \
-rendering enables optimizations that simply aren't possible without it.  """
-depends = "$auto"
-section = "rust"
-priority = "optional"
-assets = [
-    ["../target/release/alacritty", "usr/bin/", "755"],
-    ["../extra/linux/Alacritty.desktop", "usr/share/applications/", "644"],
-    ["../extra/logo/alacritty-term.svg", "usr/share/pixmaps/Alacritty.svg", "644"],
-    ["../extra/completions/alacritty.bash", "usr/share/bash-completion/completions/alacritty", "644"],
-    ["../extra/completions/alacritty.fish", "usr/share/fish/completions/alacritty.fish", "644"],
-    ["../extra/completions/_alacritty", "usr/share/zsh/vendor-completions/_alacritty", "644"],
-]
-maintainer-scripts = "../extra/linux/debian"
```

Comment:
I think removing the prebuilt Linux binaries is pretty straight-forward, since it just supports a fraction of the platforms we support and is more likely to cause troubles than actually be beneficial.

So I think the main remaining question is if we should remove the ability to build with `cargo-deb` completely, or if local deb builds should still be allowed.

Reply:
I feel like it's fine to keep them around, since we have rpm specs around, etc, especially knowing that this part of `Cargo.toml` is unlikely to change.

Reply:
Case in point against this being unlikely to change: #3628.

There's also been a couple changes to `cargo-deb` in the past. I'm generally not super against it, but I don't think 'unlikely to change' is a great argument.

Reply:
>Case in point against this being unlikely to change

Well, it's a valid issue, since we forget man page, and it's not something new.

Reply:
Yeah, of course it's valid. But that says nothing about potential future changes that are just as valid. So I wouldn't say it's particularly unlikely to change.